### PR TITLE
Create a common config for collision checking 

### PR DESCRIPTION
**Pull request by mpowelson**
Will Address #427

### CODE REVIEW

**Comment by mpowelson** [comment]
@Levi-Armstrong I'd like to get your thoughts on this and the comments in #427.

**Comment by mpowelson** [comment]
@Levi-Armstrong Any idea offhand what is going on with descartes?

**Comment by mpowelson** [comment]
Oh, I bet I have to cast some things to FloatType

**Comment by mpowelson** [comment]
@Levi-Armstrong if this passes it is ready for review. I have not fully converted the planners to use it, but I think that might come after #443

File: tesseract/tesseract_common/include/tesseract_common/types.h [hunk 1]
```diff
@@ -98,5 +98,14 @@ struct KinematicLimits
...
 };
 
+enum class Verbosity
+{
+  FATAL = 0,
+  ERROR = 10,
+  WARN = 20,
+  INFO = 30,
+  DEBUG = 40
+};
```
Comment (Levi-Armstrong) [inline tesseract/tesseract_common/include/tesseract_common/types.h:108]: Is this still being used?

File: tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp [hunk 2]
```diff
@@ -108,7 +108,8 @@ DescartesDefaultPlanProfile<FloatType>::DescartesDefaultPlanProfile(const tinyxm
         throw std::runtime_error("DescartesPlanProfile: EdgeCollisions: CollisionSafetyMargin is not a numeric "
                                  "values.");
 
-      tesseract_common::toNumeric<double>(coll_safety_margin_string, edge_collision_saftey_margin);
+      /** @todo Update xml*/
+      //      tesseract_common::toNumeric<double>(coll_safety_margin_string, edge_collision_saftey_margin);
```
Comment (Levi-Armstrong) [inline tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp:112]: A todo to update xml?

File: tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp [hunk 3]
```diff
@@ -123,7 +124,7 @@ DescartesDefaultPlanProfile<FloatType>::DescartesDefaultPlanProfile(const tinyxm
         throw std::runtime_error("DescartesPlanProfile: EdgeCollisions: LongestValidSegmentLength is not a numeric "
                                  "values.");
 
-      tesseract_common::toNumeric<double>(long_valid_seg_len_string, edge_longest_valid_segment_length);
+      //      tesseract_common::toNumeric<double>(long_valid_seg_len_string, edge_longest_valid_segment_length);
```
Comment (Levi-Armstrong) [inline tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp:127]: Same here.

**Review by Levi-Armstrong — COMMENTED** [review]
A few minor things, looks good.

File: tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp (hunk 2 above)
Comment (mpowelson) [reply tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp:112]: Yeah. Any chance we could either put that off until after #443 or someone else could chip in on that? Are you using the xml?

**Comment by mpowelson** [comment]
I just deleted Verbosity from tesseract_common and squashed everything

File: tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp (hunk 2 above)
Comment (Levi-Armstrong) [reply tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp:112]: I don't think it is currently being used, but I will look.

**Comment by mpowelson** [comment]
Added the todos for the XML

**Comment by mpowelson** [comment]
@Levi-Armstrong Ok with me merging?

**Review by Levi-Armstrong — APPROVED** [review]

**Comment by Levi-Armstrong** [comment]
> @Levi-Armstrong Ok with me merging?

Sure

**Pull request merged**